docs/coroutine-context-and-dispatchers.md

```diff
@@ -597,6 +582,9 @@ Destroying activity!
 As you can see, only the first two coroutines print a message and the others are cancelled 
 by a single invocation of `job.cancel()` in `Activity.destroy()`.
 
+> Note, that Android has first-party support for coroutine scope in all entities with the lifecycle.
+See [the corresponding documentation](https://developer.android.com/topic/libraries/architecture/coroutines#lifecyclescope).
```

Comment:
Android's LifecycleScope is only available in version `androidx.lifecycle:lifecycle-runtime-ktx:2.2.0-alpha01`, maybe it's a bit early to add this link here?

Reply:
The link seems pretty stable to me and we are not referencing a specific version here, either.

Reply:
@guelo There's been the stable 2.2.0 version released since then, so it's actually quite late.
You can see all AndroidX updates here: https://developer.android.com/jetpack/androidx/versions